src/frontend/src/lib/components/loaders/Loader.svelte

```diff
@@ -129,7 +129,9 @@
 		<div in:fade={{ delay: 0, duration: 250 }}>
 			<Modal testId={LOADER_MODAL}>
 				<div class="stretch">
-					<ImgBanner src={banner} />
+					<div class="relative mb-8 block md:h-40">
+						<ImgBanner src={banner} styleClass="h-full md:absolute aspect-auto" />
```

Comment:
is `aspect-auto` needed? i thought it was by default

Reply:
Yes because `ImgBanner` now is per default set to an aspect ratio 16-9.

Reply:
If I don't do so it defeats the absolute positioning and spacing.